doc-content/enterprise-only/release-notes/rn-7.13.1-known-issues-ref.adoc

```diff
@@ -0,0 +1,89 @@
+[id='rn-7.13.1-known-issues-ref']
+= Known issues in {PRODUCT} {ENTERPRISE_VERSION_LONG}
+
+This section lists known issues with {PRODUCT} {ENTERPRISE_VERSION_LONG}.
+
+== {CENTRAL}
+
+.Unable deploy {CENTRAL} using JDK version 11.0.16 [https://issues.redhat.com/browse/RHPAM-4497[RHPAM-4497]]
+
+Issue: It is not possible to deploy {CENTRAL} using JDK version 11.0.16.
+
+Actual result: It is not possible to deploy {CENTRAL}.
```

Comment:
[Question] - Is there feedback to the user when {CENTRAL] deployment fails due to this?